src/platform_impl/android/mod.rs

```diff
@@ -258,6 +256,11 @@ impl<T: 'static> EventLoop<T> {
                     break 'event_loop;
                 }
                 ControlFlow::Poll => {
+                    self.first_event = poll(
```

Comment:
Not polling here was an oversight in the initial implementation. It is indeed needed to support `ControlFlow::Poll`.